management/src/main/java/io/micronaut/management/endpoint/beans/BeansEndpoint.java

```diff
@@ -52,13 +52,12 @@ public BeansEndpoint(BeanContext beanContext, BeanDefinitionDataCollector beanDe
      * @return A {@link org.reactivestreams.Publisher} with the beans
      */
     @Read
-    public Single getBeans() {
+    public Publisher<?> getBeans() {
```

Comment:
This needs to be annotated with `@SingleResult` otherwise it will now return transfer coding chunked which would be a change in behaviour

Reply:
done via https://github.com/micronaut-projects/micronaut-core/pull/5583/commits/996aa4df47e0ea70c30abb1e5a619b22bc1096f0